masterlist.yaml

```diff
@@ -2391,13 +2391,65 @@ plugins:
       - crc: 0xB0489B02
         util: 'SSEEdit v4.0.3h (Hotfix 1)'
 
-  - name: 'ArteFake.esp'
+  - name: '(Awesome Artifacts - )?ArteFake(s)?( - )?(Balance|Legacy|RoM|UIT|USSEP|ZIA)?( Patch)?\.esp'
     url:
       - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/41254/'
         name: 'ArteFakes - Unique Artifacts Replacer'
+  - name: 'ArteFake.esp'
     msg:
       - <<: *patch3rdParty_RPC
         condition: 'active("Requiem.esp") and not active("Requiem - ArteFakes.esp")'
+      - <<: *patchProvided
+        subs: [ 'Reliquary of Myth - Artifact Overhaul' ]
+        condition: 'active("Reliquary of Myth.esp") and not active("ArteFake - RoM Patch.esp")'
+    tag:
+      - Actors.ACBS
+      - Actors.DeathItem
+      - Enchantments
+      - Text
```

Comment:
Missed this, but, as you stated, `Text` is already in the description of both plugins:
```suggestion
```